modules/photons_control/planner/plans.py

```diff
@@ -816,6 +821,102 @@ async def info(self):
             return self.firmware
 
 
+@a_plan("hev_status")
+class HevStatusPlan(Plan):
+    """
+    Returns the current and previous HEV cycle run info for a device. The duration,
+    remaining and power_off fields are only included if there is an HEV cycle currently
+    running.
+
+        {
```

Comment:
I feel your comment doesn't match the {"current": <current>, "last": <last>}

Reply:
Whoops, forgot to update the comment after I changed the plan. Fixed!

Reply:
thanks